cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -72,6 +72,7 @@ type initramfsMountsSuite struct {
 	kernelr2 snap.PlaceInfo
```

Comment:
do we have a test where we kept gadget unset for run mode and we check we don't fail?

Reply:
Added now